### DISCUSSION TITLE
how to install without docker?

### DISCUSSION THREAD

**Discussion by gstlouisgit** [girish17/op-mattermost#43] [Q&A] [answered]
Is there the base package for this so I can simply upload it through mattermost in the plugin section?

**Comment by girish17** [comment]
You may use [this](https://github.com/girish17/op-mattermost-plugin) Mattermost plugin. Please note that this plugin is work in progress and does not support all the features available in op-mattermost docker version.

**Comment by gstlouisgit** [comment]
@girish17 
Thanks for that.  However there isn't much to go on for install.  I moved the content in the plugin folder, enabled it as I could see it in the plugin management.  Now what?

**Comment by girish17** [comment, marked as answer]
To deploy the Mattermost plugin, please follow the instructions here - https://developers.mattermost.com/integrate/plugins/developer-setup/

As you might notice, the plugin source code must be built and then deployed to the plugins folder in mattermost-server.
